### PR TITLE
A new permission for MOTECH

### PR DESCRIPTION
**Pull request by kaapstorm**
This change allows a web user to manage integration for a project space.

new_permission

**NOTE**: Resending and cancelling repeat records currently require the `edit_web_users` permission. This remains unchanged.

new_project_settings_section project_settings_for_role

Integration-related project settings are moved into their own section. Web users in the new role can only see "My Timezone" and Integration settings.

@dannyroberts cc @calellowitz

### CODE REVIEW

**Comment by kaapstorm** [comment]
:white_check_mark: I know everyone is really busy today, but if you can eyeball this, it will unblock FGH's testing tomorrow. (They would like to be able to see their OpenMRS configuration.)

File: corehq/apps/users/templates/users/web_users.html [hunk 1]
```diff
@@ -312,6 +312,12 @@ <h3 class="panel-title">
                                 data-content="{% trans "Create, update, and delete Locations in the Organization Hierarchy." %}"></span>
                         </th>
                         <th>
+                          {% trans "Edit MOTECH Configuration" %}
+                          <span class="hq-help-template"
+                                data-title="{% trans 'Edit MOTECH Configuration' %}"
+                                data-content="{% trans 'Manage integration with other systems.' %}"></span>
+                        </th>
```
Comment (dannyroberts) [inline corehq/apps/users/templates/users/web_users.html]: Can you make this row only show up if you have the openmrs flag set? We shouldn't expose a checkbox that does nothing (whether checked or unchecked) for the vast majority of users.

File: corehq/motech/dhis2/view.py [hunk 2]
```diff
@@ -122,13 +115,9 @@ def get_queryset(self):
...
 
-    @method_decorator(domain_admin_required)
-    def dispatch(self, request, *args, **kwargs):
-        if not toggles.DHIS2_INTEGRATION.enabled(request.domain):
-            raise Http404()
-        return super(Dhis2LogListView, self).dispatch(request, *args, **kwargs)
-
 
+@method_decorator(require_permission(Permissions.edit_motech), name='dispatch')
+@method_decorator(toggles.DHIS2_INTEGRATION.required_decorator(), name='dispatch')
```
Comment (calellowitz) [inline corehq/motech/dhis2/view.py]: didnt realize you could do this with `method_decorator` thats cool

**Review by calellowitz — APPROVED** [review]
Looks good to me

File: corehq/apps/users/templates/users/web_users.html [hunk 3]
```diff
@@ -311,12 +311,14 @@ <h3 class="panel-title">
                                 data-title="{% trans "Edit Locations"%}"
                                 data-content="{% trans "Create, update, and delete Locations in the Organization Hierarchy." %}"></span>
                         </th>
+                        {% if show_integration %}
                         <th>
-                          {% trans "Edit MOTECH Configuration" %}
+                          {% trans "Edit Integration" %}
```
Comment (kaapstorm) [inline corehq/apps/users/templates/users/web_users.html]: ("Edit MOTECH Configuration" looked awkwardly long as a column heading.)

**Pull request merged**